bundle/manifests/cert-manager.clusterserviceversion.yaml

```diff
@@ -0,0 +1,152 @@
+                    args:
+                      - start
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.annotations['olm.targetNamespaces']
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: cert-manager-operator
+                    image: docker.io/slaskawi/cert-manager-operator:latest
+                    imagePullPolicy: Always
+                    name: cert-manager-operator
+                    resources: { }
+                serviceAccountName: cert-manager-operator
+      permissions:
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - events
+              verbs:
+                - list
+                - get
+                - create
+                - patch
+                - update
+                - watch
+                - delete
+            - apiGroups:
+                - apps
+              resources:
+                - deployments
+              verbs:
+                - list
+                - get
+                - create
+                - update
+                - watch
+            - apiGroups:
+                - apiextensions.k8s.io
+              resources:
+                - customresourcedefinitions
+              verbs:
+                - list
+                - get
+                - create
+                - patch
+                - update
+                - watch
+                - delete
+            - apiGroups:
+                - rbac.authorization.k8s.io
+              resources:
+                - clusterrole
+              verbs:
+                - list
+                - get
+                - create
+                - watch
+                - delete
+            - apiGroups:
+                - monitoring.coreos.com
+              resources:
+                - servicemonitors
+                - prometheusrules
+              verbs:
+                - list
+                - get
+                - create
+                - update
+                - watch
+          serviceAccountName: cert-manager-operator
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: false
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  keywords:
+    - TLS
+    - secrets
+    - security
+    - certificates
+  links:
+    - name: Documentation
+      url: https://github.com/openshift/cert-manager-operator/blob/master/README.md
+  maintainers:
+    - email: cert-manager-maintainers@googlegroups.com
+      name: The cert-manager maintainers
+  maturity: stable
+  minKubeVersion: 1.16.0
```

Comment:
Maybe 1.21?

Reply:
let's align this with openshift, hence 1.22. 4.10 will be 1.23 so we can bump once the rebase is done. we cannot express a dependency to openshift right?

Reply:
The OpenShift dependency is defined elsewhere. Here we configure Kube only.